src/sdk/accountspace_nb.js

```diff
@@ -0,0 +1,261 @@
+            },
+            account: requesting_account,
+        };
+        // CORE CHANGES PENDING - START
+        const iam_account = await account_util.create_account(req);
+        // CORE CHANGES PENDING - END
+
+        // TODO : Clean account cache
+        // TODO : Send Event
+        return {
+            // TODO : PATH Missing
+            iam_path: requesting_account.iam_path || IAM_DEFAULT_PATH,
+            username: params.username,
+            user_id: iam_account.id,
+            arn: iam_arn,
+            create_date: iam_account.create_date,
+        };
+
+    }
+
+    async get_user(params, account_sdk) {
+        const action = IAM_ACTIONS.GET_USER;
+        const requesting_account = system_store.get_account_by_email(account_sdk.requesting_account.email);
+        const account_name = new SensitiveString(`${params.username}:${requesting_account.name.unwrap()}`);
+        const requested_account = system_store.get_account_by_email(account_name);
+        account_util._check_if_requesting_account_is_root_account(action, requesting_account,
+                { username: params.username, iam_path: params.iam_path });
+        account_util._check_if_account_exists(action, params.username, requesting_account);
+        account_util._check_if_requested_account_is_root_account_or_IAM_user(action, requesting_account, requested_account);
+        // CORE CHANGES PENDING - START
+        //const root_account = system_store.get_account_by_email(requesting_account.email);
+        // CORE CHANGES PENDING - END
+        account_util._check_if_requested_is_owned_by_root_account(action, requesting_account, requested_account);
+        const reply = {
+            user_id: requested_account._id.toString(),
+            // TODO : IAM PATH
+            iam_path: requested_account.iam_path || IAM_DEFAULT_PATH,
+            username: requested_account.name.unwrap().split(IAM_SPLIT_CHARACTERS)[0],
+            arn: requested_account.iam_arn,
+            create_date: new Date(requested_account.last_update),
+            // TODO: Dates missing : GAP
+            password_last_used: new Date(),
+        };
+        return reply;
+    }
+
+    async update_user(params, account_sdk) {
+        const action = IAM_ACTIONS.UPDATE_USER;
+        const requesting_account = system_store.get_account_by_email(account_sdk.requesting_account.email);
+        const account_name = new SensitiveString(`${params.username}:${requesting_account.name.unwrap()}`);
+        account_util._check_if_requesting_account_is_root_account(action, requesting_account,
+                { username: params.username, iam_path: params.iam_path });
+        account_util._check_if_account_exists(action, params.username, requesting_account);
+        const requested_account = system_store.get_account_by_email(account_name);
+        let new_iam_path = requested_account.iam_path;
+        let new_user_name = requested_account.name.unwrap();
```

Comment:
I think it is confusing to set the variables `new_iam_path` and `new_user_name` with what we have already.